docs/instructions.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>UHub - Installation Instructions</title>
+  <link href="./css/main.css" type="text/css" rel="stylesheet">
+</head>
+<body>
+  <img id="logo" src="./assets/uhub_logo_800x.png" alt="UHub">
+  <h1 class="tagline">Installation Instructions</h1>
+  <h2>How to install UHub</h2>
+  <p>
+    UHub is currently in early-access beta and is available for download on both iOS and Android devices.
+    <br/><br/>
+    While the app is not yet listed on the Apple App Store or Google Play Store,
+    you can download the app by following the instructions below.
+    <h3>Table of Contents</h3>
+    <ul>
+      <li><a href="#installing-ios">Installing on iOS</a></li>
+      <li><a href="#installing-android">Installing on Android</a></li>
+    </ul>
+    <h2 id="installing-ios">Installing on iOS</h2>
+    <p>
+      [TODO: Add instructions for installing on iOS]
```

Comment:
@rahulniraula2 Once the PR is merged, Do you mind filling in the installation instructions for TestFlight in a new PR since you're more familiar with it.

Reply:
Got it! will do